app/src/main/java/com/teamltt/carcare/activity/HomeActivity.java

```diff
@@ -146,27 +147,6 @@ public boolean onCreateOptionsMenu(Menu menu) {
         return true;
     }
 
-    protected void goToDemo(View view) {
-        Intent intent = new Intent(this, DemoActivity.class);
-        startActivity(intent);
-    }
-    protected void goToStatic(View view) {
-        Intent intent = new Intent(this, DemoActivity.class);
-        startActivity(intent);
-    }
-    protected void goToDynamic(View view) {
-        Intent intent = new Intent(this, DemoActivity.class);
-        startActivity(intent);
-    }
-
-    /*protected void openDrawer(View view) {
-        if (drawer.isDrawerOpen(findViewById(android.R.id.home))) {
-            drawer.closeDrawer(Gravity.LEFT);
-        }
-        else {
-            drawer.openDrawer(Gravity.RIGHT);
-        }
-    }*/
     public boolean onOptionsItemSelected(MenuItem item) {
```

Comment:
An activity should only handle menu options that are specific to it. BaseActivity only handles Up navigation, and HomeActivity handles links to car info/static page/dynamic page/etc.
However this is largely irrelevant because only the home page has a settings menu at all at this point in the wireframe. I only kept the menu there so we could access activities without the search feature which doesn't exist yet.